about/list_of_features.rst

```diff
@@ -0,0 +1,514 @@
+
+**Particles:**
+
+- GPU-based particles with support for custom particle shaders.
+- CPU-based particles.
+
+**Post-processing:**
+
+- Tonemapping (Linear, Reinhard, Filmic, ACES).
+- Automatic exposure adjustments based on viewport brightness.
+- Near and far depth of field with adjustable bokeh simulation.
+- Screen-space ambient occlusion at half or full resolution.
+- Glow/bloom with optional bicubic upscaling and several blend modes available:
+  Screen, Soft Light, Add, Replace, Mix.
+- Color correction using an one-dimensional ramp.
+- Roughness limiter to reduce the impact of specular aliasing.
+- Brightness, contrast and saturation adjustments.
+
+**Texture filtering:**
+
+- Nearest, bilinear, trilinear or anisotropic filtering.
+- Filtering options are defined on a per-use basis, not a per-texture basis.
+
+**Texture compression:**
+
+- Basis Universal (slow, but results in smaller files).
+- BPTC for high-quality compression (not supported on macOS).
+- ETC2 (not supported on macOS).
+- S3TC (not supported on mobile/Web platforms).
+
+**Anti-aliasing:**
+
+- Fast approximate antialiasing (FXAA).
+- Multi-sample antialiasing (MSAA).
+
+Most of these effects can be adjusted for better performance or to further
+improve quality. This can be helpful when using Godot for offline rendering.
+
+3D tools
+^^^^^^^^
+
+- Built-in meshes: cube, cylinder/cone, (hemi)sphere, prism, plane, quad.
+- Tools for :ref:`procedural geometry generation <doc_procedural_geometry>`.
+- :ref:`Constructive solid geometry <doc_csg_tools>` (intended for prototyping).
+- Path3D node to represent a path in 3D space.
+   - Can be drawn in the editor or generated procedurally.
+   - PathFollow3D node to make nodes follow a Path3D.
+- 3D geometry helper class.
+
+3D physics
+^^^^^^^^^^
+
+**Physics bodies:**
+
+- Static bodies.
+- Rigid bodies.
+- Kinematic bodies.
+- Vehicle bodies (intended for arcade physics, not simulation).
+- Joints.
+- Soft bodies.
+- Ragdolls.
+- Areas to detect bodies entering or leaving it.
+
+**Collision detection:**
+
+- Built-in shapes: infinite plane, cuboid, sphere, capsule, cylinder (only with
+  Bullet physics).
+- Generate triangle collision shapes for any mesh from the editor.
+- Generate one or several convex collision shapes for any mesh from the editor.
+
+Shaders
+^^^^^^^
+
+- *2D:* Custom vertex, fragment, and light shaders.
+- *3D:* Custom vertex, fragment, light, and sky shaders.
+- Text-based shaders using a `shader language inspired by GLSL <doc_shading_language>`.
+- Visual shader editor.
+   - Support for visual shader plugins.
+
+Scripting
+^^^^^^^^^
+
+**General:**
+
+- Signals and groups.
+- Many built-in datatypes to ease the development process:
+   - 2D and 3D vectors, available in floating-point and integer form.
+   - 2D and 3D transforms.
+   - 2D rectangles.
+   - 3D axis-aligned bounding boxes (AABBs).
+   - 3D infinite planes.
+   - Colors.
+   - Quaternions.
```

Comment:
I feel like this list is not very necessary, and/or can be compacted. We aren't listing every single node type, so there is no need to list every single math type. I think this list should be replaced with "Many 2D and 3D linear algebra data types such as vectors and transforms". Also, there are a few things missing when it comes to general scripting features. I think it should be something like:

```rst
**General:**

- Object-oriented design pattern with scripts extending nodes.
- Signals and groups for communicating between scripts.
- Support for cross-language scripting.
- Many 2D and 3D linear algebra data types such as vectors and transforms.
```

Maybe also link https://docs.godotengine.org/en/latest/getting_started/scripting/cross_language_scripting.html